src/AI/Muriel.java

```diff
@@ -841,41 +848,41 @@ private void queueUnitProductionActions(GameMap gameMap)
       // general; we want units that are good against more than just enemyToCounter, if possible.
       log("  Initial scoring:");
       Queue<ModelValuePair> counterScores = new PriorityQueue<ModelValuePair>(counters.size());
-      for(UnitModel counter : counters)
+      for(ModelForCO counter : counters)
       {
         int score = 0;
         for( Commander enemyCo : unitLists.keySet() )
         {
           for( Unit enemyUnit : unitLists.get(enemyCo) )
           {
-            UnitMatchupAndMetaInfo umami = getUnitMatchupInfo(counter, enemyUnit.model);
+            UnitMatchupAndMetaInfo umami = getUnitMatchupInfo(counter, new ModelForCO(enemyUnit.CO, enemyUnit.model));
             if( useDamageRatio )
             {
               if( umami.damageRatio >= 1.0 ) score++; // Plus one if it's worth building.
               if( umami.damageRatio >= 1.5 ) score++; // An extra bump in score if they are very good vs this type.
-              if( umami.damageRatio < 0.8 ) score--; // Discount if my counter is countered.
+              if( umami.damageRatio < 0.8 ) score--; // Discount if myArmy counter is countered.
```

Comment:
Did you do a find/replace for "my" -> "myArmy"? There are a few of these.

Reply:
Seems likely.
Wasn't intentional.